style.css

```diff
@@ -0,0 +1,105 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
+
+.container-toolbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  padding: 0 24px;
+}
+
+.toggle li {
+  list-style: none;
+}
+
+.hamburger {
+  cursor: pointer;
+  padding: 5px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #28352f;
+}
+
+.toolbar-title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  padding-top: 13px;
+  color: #28352f;
+  text-decoration: none;
+}
+
+.toggle {
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border-radius: 8px;
+  flex-direction: row;
+  padding-top: 10px;
+}
+
+body {
+  width: 375px !important;
+  height: 6715px;
+  text-align: center;
+  justify-content: space-around;
+  background-color: #e5e5e5;
+}
+
+.headline-name {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  padding-top: 30px;
+  padding-bottom: 0;
+  margin-bottom: 0;
+}
+
+.headline-job {
+  font-family: 'inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  color: #36b37e;
+  line-height: 52px;
+  margin: 0;
+  padding-top: 0;
+}
```

Comment:
[Optional[I think Your intro header is looking a bit different from the original design in `font-weight`🤔

your webpage
<img width="268" alt="yourHey" src="https://user-images.githubusercontent.com/64643995/212259227-3a5fbb12-93e2-4c39-94ab-1742163bab31.png">

the design
<img width="243" alt="figmaHey" src="https://user-images.githubusercontent.com/64643995/212259340-f5563550-1110-40ad-8f61-d98ab84fbd1b.png">

it is a compulsory requirement that you stick to the design, I think the previous reviewer didn't take note of it.
kindly make the adjustments where appropriate😉